Integrations/MongoDB/MongoDB.yml

```diff
@@ -0,0 +1,195 @@
+category: Database
+commonfields:
+  id: MongoDB
+  version: -1
+configuration:
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- additionalinfo: A comma separated list of urls with port.
+  display: Server URLs with port (host1.com:27017,host2.com:27017)
+  name: urls
+  required: true
+  type: 0
+- display: Database
+  name: database
+  required: true
+  type: 0
+- display: Trust any certificate (not secure)
+  hidden: false
+  name: insecure
+  required: false
+  type: 8
+description: Use MongoDB to search and query entries
+display: MongoDB
+name: MongoDB
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: Name of the collection do get entry from.
+      isArray: false
+      name: collection
+      required: true
+      secret: false
+    - default: false
+      description: An ObjectID to get.
+      isArray: true
+      name: object_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Get an entry from database by ID
+    execution: false
+    name: mongodb-get-entry-by-id
+    outputs:
+    - contextPath: MongoDB._id
```

Comment:
```suggestion
    - contextPath: MongoDB.ID
```

Reply:
i wanted to stary consistent on how it looks in MongoDB.

Reply:
stay consistent with demisto